framework/lib/components/combo-picker/combo-picker.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import { Box, Button, InputGroup, InputRightElement, NumberInputField, useDisclosure } from '@chakra-ui/react'
+import { ComboPickerProps } from './types'
+import { Select } from '../select'
+import { NumberInput } from '../number-input'
+
+/**
+ * @see ComboPicker
+ * @see {@link https://northlight.dev/reference/combo-picker}
+ */
+export const ComboPicker = ({
+  onChange,
+  options,
+  size,
+  value,
+  placeholder,
+  isDisabled,
+  ...rest
+}: ComboPickerProps) => {
+  const { isOpen, onToggle, onClose } = useDisclosure()
+  const [ inputValue, setInputValue ] = useState(value ? value.input : 0)
```

Comment:
I believe NumberInput does not support placeholder so I think it feels weird to have it completely blank with nothing, this would also indicate clearly that a number is expected, but what do you think?

Reply:
fair enough 👍